components/DappFormFieldsProductImage.vue

```diff
@@ -85,4 +91,24 @@ export default {
   margin-top: 0.5rem;
   margin-bottom: 1rem;
 }
+
+.preview {
+  height: 0;
+  padding-top: 52.5%;
+  position: relative;
+  &-box {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    &-image {
+      width: 100%;
+      max-width: 100%;
+      display: block;
+      height: 100%;
```

Comment:
I set the image to `width: 100%; max-width: 100%` to ensure that it always fills the full width. `display: block` ensures no default white space below the image. `height: 100%` makes the image scrunch. To undo the scrunch, remove the height and it will only show the top.